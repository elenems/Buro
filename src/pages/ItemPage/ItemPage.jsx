import React from "react";
import Header from "../../blocks/Header";
import Container from "@material-ui/core/Container";
import Grid from "@material-ui/core/Grid";
import Item from '../../blocks/Item';

export default function ItemPage() {
  return (
    <Container fixed>
      <Grid container spacing={6}>
        <Header />
        <Item />
      </Grid>
    </Container>
  );
}
