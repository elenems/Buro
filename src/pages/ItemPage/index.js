import ItemPage from './ItemPage.jsx';
export default ItemPage;