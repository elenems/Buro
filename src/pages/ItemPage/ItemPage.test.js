import React from "react";
import { shallow } from "enzyme";
import ItemPage from "../ItemPage";

describe("ItemPage", () => {
  it("Matches snapshot", () => {
    const wrapper = shallow(<ItemPage />).debug();
    expect(wrapper).toMatchSnapshot();
  });
});
