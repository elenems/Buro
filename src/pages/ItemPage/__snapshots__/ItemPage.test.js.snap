// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemPage Matches snapshot 1`] = `
"<WithStyles(ForwardRef(Container)) fixed={true}>
  <WithStyles(ForwardRef(Grid)) justify=\\"flex-start\\" direction=\\"column\\" container={true} spacing={6}>
    <withRouter(Header) />
    <withRouter(Item) />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Container))>"
`;
